### ISSUE TITLE
Tug 0.6.0.0 - StrictInputFilter validation failure with Windows 10 LCM client

### ISSUE THREAD

**Issue by bbollard** [PowerShellOrg/tug#80]
Hello,
I am evaluating your pre-packaged version 0.6.0.0 (the newer package is still broken). I am working getting my Windows 10 LCM talking with the Tug Server. Registration seems to go OK. But The GetDscAction action is failing due to strict input filter checking. 

Is there any guidance on how I should proceed here? I can't seem to find an application setting that would turn strict checking off. Since it is the Windows 10 LCM talking to Tug is this a bug in how Tug is supposed to work? 

Thank you for your assistance. I haven't seen much activity on here lately so I'm beginning to fear this project has hit the end of the road -- I do hope that is not the case.

Regards,
Brian

```

> 2017-09-15 15:05:19.1498|0|WARN|Tug.Server.Filters.StrictInputFilter|Re-serialized representation does not match actual input
> 2017-09-15 15:05:19.1498|0|TRACE|Tug.Server.Filters.StrictInputFilter|Expected:  {"ClientStatus":[{"ConfigurationName":"BaseWindows","Checksum":"","ChecksumAlgorithm":"SHA-256"}]}
> 2017-09-15 15:05:19.1498|0|TRACE|Tug.Server.Filters.StrictInputFilter|Actual:  {"ClientStatus":[{"Checksum":"","ConfigurationName":"BaseWindows","ChecksumAlgorithm":"SHA-256"}]}

```

**Comment by ebekker** [comment]
That's interesting, I don't think I've actually done any testing with the Win10 LCM, only Win2016 Server LCM and the test client.  I'll need to check this out.

**Comment by ebekker** [comment]
No, this project is not dead, I just have a lull during Q2 and Q3 every year due to work-related activities, but as that comes to completion I'll be resuming my activity here.  Also, because the direction of PSCore was up in the air for a while for many items, I was waiting for that and .NET Core 2.0 to stabilize so that we can take some reliable dependencies.

**Comment by bbollard** [comment]
Sounds great. Eliminating the StrictInputFilter validation entirely does allow a Windows 10 LCM to successfully communicate with Tug and subsequently enact the local configuration.

**Comment by mnothh** [comment]
I seem to be running into the same issue here.  Is there a version with the patch applied available or instructions for making this tweak and rebuilding myself?

Thanks,
-Mike
